omod/src/main/webapp/pages/myaccount/changeDefaults.gsp

```diff
@@ -1,30 +1,32 @@
 <%
-    ui.decorateWith("appui", "standardEmrPage", ["title": "Change User Defaults"])
```

Comment:
Did you intentionally change from "Change User Defaults" to "My Languages"?

Reply:
Yes. Just look at the ticket description :)
 https://issues.openmrs.org/browse/RA-995

Reply:
The one that i see in the ticket description is "Default Settings" changed to "My Languages"

Reply:
Ooohh sorry, I thought since it directs to a "My Languages" page, it was worth such wording. Let me update it :)

Reply:
But wait, have you looked at the second point in the description?

Reply:
What sayeth it?

Reply:
<ol>
<li>On that page:
<ul><li>the title should be "My Languages"</li>
<li>the first field should be "Primary Language" and it should have descriptive help text "the User Interface will be displayed in this language"</li>
</ul>
</li>
</ol>

Reply:
I'm wrong at understanding this second point? Or I read the description poorly?

Reply:
What is the current title, which should be My Languages?

Reply:
Its "Change User Defaults"

Reply:
Can you share a screenshot that has it?

Reply:
Just simply load this page https://demo.openmrs.org/openmrs/adminui/myaccount/changeDefaults.page
check the title is "Change User Defaults"

Reply:
![changedefs](https://user-images.githubusercontent.com/26084581/36469643-c4db8010-169c-11e8-8653-3d42214dcbd3.png)